nss-day-cohort-48/the-worst-site-ever-fandango
# edited the css

edited the css that was added so that it does not conflict with the main css styling

##steps to test:
1. pull and serve/ code .
2. check that the home page has the navbar directly under the header and that there is no green background behind the about section
